src/modules/pivotTable/clipAxis.js

```diff
@@ -1,11 +1,29 @@
 import times from 'lodash/times'
 
-export const clipAxis = ({ position, size, step, totalCount, headerCount }) => {
+export const clipAxis = ({
+    position,
+    size,
+    step,
+    totalCount,
+    headerCount,
+    fixedColumnHeaders,
```

Comment:
`clipAxis` is generic and used for both rows and columns, should this just be `fixedHeaders` ?

Reply:
I'm not sure, what should `fixedHeaders` be then?
`headerCount` is looking at the columns only, and `count` (which is the number of rows) is then adjusted removing the number of header rows at the top of the table, but only when fixed columns is enabled.
I could rename the variable, but I think it's important to know which one of the 2 options is actually passed there.

Reply:
It turns out we don't need to change this part, I reverted the changes to this file as we discovered a bug.